bundledata.go

```diff
@@ -303,6 +318,11 @@ func (verifier *bundleDataVerifier) verifyServices() {
 		if err := verifier.verifyConstraints(svc.Constraints); err != nil {
 			verifier.addErrorf("invalid constraints %q in service %q: %v", svc.Constraints, name, err)
 		}
+		for storageName, storageConstraints := range svc.Storage {
```

Comment:
Yes, there is, thanks for picking that up. I've added validation of the storage name too, along with a test.